addons/web/static/src/views/list/list_renderer.js

```diff
@@ -233,10 +233,8 @@ export class ListRenderer extends Component {
         this.isRTL = localization.direction === "rtl";
     }
 
-    displaySaveNotification() {
-        this.notificationService.add(_t("Please save your changes first"), {
-            type: "danger",
-        });
+    async openFormView(record, newWindow) {
```

Comment:
not sure it's necessary, you could directly call this.props.onFormView in the template, like done before